Source/SteveSloth/Private/MyEnemyAttackingState.cpp

```diff
@@ -0,0 +1,69 @@
+/****************************************************************************************
+ * Copyright: SteveSloth
+ * Name: Ken Ferris
+ * Script: MyEnemyAttackingState.cpp
+ * Date: May 3, 2024
+ * Description: Enemy Attack State
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+
+#include "MyEnemyAttackingState.h"
+#include "MyEnemyBaseClass.h"
```

Comment:
This will cause circular inclusion problems, you might want to put the attack variables from the base class in this script.